packages/contracts-bedrock/snapshots/storageLayout/OPStackManager.json

```diff
@@ -14,7 +14,7 @@
     "type": "bool"
   },
   {
-    "bytes": "160",
```

Comment:
This hints that we should move the blueprints storage var to be the last one so we can more easily adjust it in the future after it's deployed, and/or have a spacer in here. I'll create a sticky for this in figjam